rust/src/quic/logger.rs

```diff
@@ -30,14 +31,40 @@ fn log_template(tx: &QuicTransaction, js: &mut JsonBuilder) -> Result<(), JsonEr
             js.set_string("ua", &String::from_utf8_lossy(&ua))?;
         }
     }
-    js.open_array("cyu")?;
-    for cyu in &tx.cyu {
-        js.start_object()?;
-        js.set_string("hash", &cyu.hash)?;
-        js.set_string("string", &cyu.string)?;
+    if tx.cyu.len() > 0 {
```

Comment:
Oh, I also added, the not logging of empty array...
Should this be part of jsonschema verifications @jasonish ?

Reply:
Probably. I don't think empty arrays should ever be logged.  And we've had cases where empty arrays were not, then were (mostly due to migration to jsonbuilder), and that would be nice to catch.

Reply:
Do you want a QA redmine ticket for this ?

Reply:
https://redmine.openinfosecfoundation.org/issues/5167